tests/integration_test/src/utils.py

```diff
@@ -333,6 +333,8 @@ def generate_test_config_yaml_for_example(
     output_yamls = []
     os.makedirs(OUTPUT_YAML_DIR, exist_ok=True)
     for job in os.listdir(example.jobs_root_dir):
+        if "base" in job:
+            continue
```

Comment:
Locally user will run "prepare_job_config.sh" that's correct.

But The automatic test stage in the integration tests will actually use "base" job directory.
So we can't add these two lines.

Reply:
Update: current integration test framework needs some modifications to support these base jobs; to be fixed in the future